pkg/morph/client/constructor.go

```diff
@@ -100,11 +100,17 @@ func New(key *ecdsa.PrivateKey, endpoint string, opts ...Option) (*Client, error
 		return nil, err
 	}
 
+	neo, err := cli.GetNativeContractHash(nativenames.Neo)
```

Comment:
Does `cli.GetNativeContractHash` actually return static value?

Reply:
Technically it can return different values. However it is not designed to behave like this. NEO, GAS, RoleManagement(designate) and Notary contracts are native and can be changed without script hash change.

Reply:
However it will be nice to get insight on this from @roman-khimov, is it legit to cache script hashes of native contracts at the application start or not.